mathesar/signals.py

```diff
@@ -0,0 +1,22 @@
+from django.db.models.signals import post_save, pre_delete
+from django.dispatch import receiver
+
+from mathesar.models import Table, Constraint
+from mathesar.reflection import reflect_new_table_constraints
+from db.constraints import drop_constraint
+
+
+@receiver(post_save, sender=Table)
+def create_new_constraints(sender, instance, created, **kwargs):
```

Comment:
I ended up removing the `pre_delete` signal because I realized that we delete constraint models in other places and we don't always automatically want to drop them. However, I did rename the function to be clearer and added a comment.